### ISSUE TITLE
Make iDOM Compatible With Nteract's VDOM

### ISSUE THREAD

**Issue by rmorshea** [idom-team/idom#92]
After #73 iDOM's spec was nearly compatible with [Nteract's VDOM](https://github.com/nteract/vdom), but VDOM still doesn't have a way to indicate `stopPropagation` or `preventDefault`.

**Comment by rmorshea** [comment]
New issue added to support `importSource`:

**Comment by rmorshea** [comment]
I kind of think we can give up on this. I'm not especially interested in remaining compatible since the two projects have mostly diverged. I kind of wonder whether I should rename "attributes" to "props" if we ignore `vdom` completely.

**Comment by rmorshea** [comment]
I think that's off the table. `vdom` hasn't been touched in several years.
